stable/graylog/templates/statefulset.yaml

```diff
@@ -169,7 +175,10 @@ spec:
                   - bash
                   - -ec
                   - |
-                    curl -XPOST -sS -u "{{ .Values.graylog.rootUsername }}:${GRAYLOG_PASSWORD_SECRET}" -H "X-Requested-By: {{ template "graylog.fullname" . }}" "localhost:9000/api/system/shutdown/shutdown"
+                    curl {{- if .Values.graylog.tls.enabled }}-k{{- end }} -XPOST -sS \
```

Comment:
I chose not to install the (possibly self-signed) CA since its talking over `localhost`, I figured ignoring root check security was fine enough.